CHANGELOG.md

```diff
@@ -4,6 +4,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 # Upcoming
 
 ### 🔄 Changed
+- An error message about recovery request mode was incorrectly logged after web-socket connect [#3426](https://github.com/GetStream/stream-chat-swift/pull/3426)
```

Comment:
It was asked by a customer

Reply:
Only because I told them that it gets fixed. Since it was logging an error, it can end up in Firebase's error reporting section (this is where they saw it).